pymc3/tests/test_parallel_sampling.py

```diff
@@ -97,6 +98,9 @@ def test_abort(mp_start_method):
 
     step = pm.CompoundStep([step1, step2])
 
+    # on Windows we cannot fork
+    if platform.system() == "Windows":
+        mp_start_method = "spawn"
```

Comment:
```suggestion
    if platform.system() == "Windows" and mp_start_method == "fork":
        return
```

Reply:
Made the change (I agree it's better) and of course now the test starting passing on Windows/fork (return before the error had a chance to happen), resulting in CI complaining because of strict xfail settings...

Reply:
Makes sense